.serena/memories/suggested_commands.md

```
@@ -1,20 +1,60 @@
-- `pnpm run test`: Run AVA tests
-
-## Project Structure
-- `src/`: Source code
-- `tests/`: Unit and integration tests
-- `packages/`: JavaScript/TypeScript modules
-- `docs/`: Documentation
-- `configs/`: Configuration files`
\ No newline at end of file
+# Development Commands
+
+## Core Development Workflow
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)